### ISSUE TITLE
What a complete joke

### ISSUE THREAD

**Issue by joekendal** [ARKInvest/ARK-Invest-Tesla-Valuation-Model#26]
So Cathie's bull case is a situation where TSLA has a market cap of 20% of US GDP! 🤯

For a car company that potentially made more from pumping Bitcoin than selling cars you have to wonder what Cathie Wood is smoking. 

1. Tesla has inferior self-driving car technology. They are not the leaders. They have the data but they don't have the algorithms.

2. They have no ride-hailing business. If they were to create one, they already have competition so to assume they would instantly win the whole market is a joke.

3. The valuation of the insurance business is ridiculous. I wonder if anyone at Ark has any financial literacy.

4. The CEO is more interested in promoting a stupid cryptocurrency backed by a dog meme.

5. The CEO doesn't want to sell cars but would rather pursue his real mission which is to colonise a different planet. The lack of focus is apparent.

6. Other car companies are able to copy their technology and we're already seeing **a lot** of new competition from industry veterans and startups.

7. China won't allow a US company to dominate that market. Look at what's happening already.

8. Tesla is a massive bubble backed by unsophisticated retail speculators. The correlation between stimmy check issuance and Tesla call option volume is a joke.

9. Look who else is short TSLA - Burry and Chanos. They have a track record that speaks for itself. Cathie has one year of interesting returns. All supported by Jerome Powell.

10. Once you adjust for risk, Ark has ZERO alpha and is just a leveraged NASDAQ beta play.

11. Yields will continue to rise. No amount of QE will stop the bubble from imploding and Ark investors will be left holding the bag.

12. Even if I'm wrong about 11 and bond yields stay below 3%, TSLA still has a P/E ratio of over 1000. An insane amount for any company and especially with slowing growth.

13. Look at the schmucks who are buying this stock at these prices. 14 year old TikTok teenagers. We saw this in 2000 with Pets.com, we saw this in 1929 with the shoeshine boys giving stock tips. 

14. There has **never** in history been a bull market that has gone on for this long that HASN'T resulted in a 55% or more **CRASH** in the S&P. This one will rival '29.

15. Perhaps one of the only metric you should have used in your model was the $1m per car one.

Tell me the difference between Cathie's portfolio and a 12 year old with a Robinhood account.

TSLA is going to $300 📉 along with all the other bubble stocks in Ark's ETFs.

**Comment by simacastel** [comment]
> So Cathie's bull case is a situation where TSLA has a market cap of 20% of US GDP! 🤯
>

No, TSLA in 2025 will be worth 20% of US GDP in 2020. You have to factor in US growth as well.

> For a car company that potentially made more from pumping Bitcoin than selling cars you have to wonder what Cathie Wood is smoking.
> 

Needless FUD.

> 1. Tesla has inferior self-driving car technology. They are not the leaders. They have the data but they don't have the algorithms.

They have the only self-driving tech that can scale. Waymo and Baidu are massively behind.

> 2. They have no ride-hailing business. If they were to create one, they already have competition so to assume they would instantly win the whole market is a joke.

You can adjust for 0% ride hailing. It doesn't impact the bear case that much.

> 3. The valuation of the insurance business is ridiculous. I wonder if anyone at Ark has any financial literacy.

More FUD. Going to need actual numbers.

> 4. The CEO is more interested in promoting a stupid cryptocurrency backed by a dog meme.

Irrelevant FUD.

> 5. The CEO doesn't want to sell cars but would rather pursue his real mission which is to colonise a different planet. The lack of focus is apparent.

Irrelevant FUD.

> 6. Other car companies are able to copy their technology and we're already seeing **a lot** of new competition from industry veterans and startups.

You can copy the tech, but you can't copy the engineering, production, data, supply chain, or business model. Where's the competition?

> 7. China won't allow a US company to dominate that market. Look at what's happening already.

Speculative FUD.

> 8. Tesla is a massive bubble backed by unsophisticated retail speculators. The correlation between stimmy check issuance and Tesla call option volume is a joke.

Irrelevant ad hom.

> 9. Look who else is short TSLA - Burry and Chanos. They have a track record that speaks for itself. Cathie has one year of interesting returns. All supported by Jerome Powell.

Cathie has decades of interesting returns if you bothered to follow her. She also called Amazon in 2002 and bought in at $5Bn market cap.

Burry, on the other hand, had poor returns since 2008. 

> 10. Once you adjust for risk, Ark has ZERO alpha and is just a leveraged NASDAQ beta play.

Meaningless statement with meaningless buzzwords.

> 11. Yields will continue to rise. No amount of QE will stop the bubble from imploding and Ark investors will be left holding the bag.

Yields have no impact on TSLA fundamentals. They already have enough cash to weather the storm. They're not in a credit crunch like they were in 2008, and even then they survived.

> 12. Even if I'm wrong about 11 and bond yields stay below 3%, TSLA still has a P/E ratio of over 1000. An insane amount for any company and especially with slowing growth.

Amazon had a P/E ratio of 3000. P/E doesn't mean anything if you knew how business accounting works.

> 13. Look at the schmucks who are buying this stock at these prices. 14 year old TikTok teenagers. We saw this in 2000 with Pets.com, we saw this in 1929 with the shoeshine boys giving stock tips.

Ad Homs. 

> 14. There has **never** in history been a bull market that has gone on for this long that HASN'T resulted in a 55% or more **CRASH** in the S&P. This one will rival '29.

Market timing doesn't work. Sorry.

> 15. Perhaps one of the only metric you should have used in your model was the $1m per car one.
> 

??? Is this sarcasm? Irrelevant ad homs.

> Tell me the difference between Cathie's portfolio and a 12 year old with a Robinhood account.
> 

12 year olds can't open Robinhood accounts.

> TSLA is going to $300 📉 along with all the other bubble stocks in Ark's ETFs.

You should short it then. Good luck!

Posts like these make me happy. It means I'm still early into TSLA.

Have a nice day!

**Comment by kaovilai** [comment]
If they make another split it might hit 300 ;)

**Comment by joekendal** [comment]
> > So Cathie's bull case is a situation where TSLA has a market cap of 20% of US GDP! 🤯
> 
> No, TSLA in 2025 will be worth 20% of US GDP in 2020. You have to factor in US growth as well.
> 
Both cases are ridiculous
> > For a car company that potentially made more from pumping Bitcoin than selling cars you have to wonder what Cathie Wood is smoking.
> 
> Needless FUD.
> 
Agreed
> > 1. Tesla has inferior self-driving car technology. They are not the leaders. They have the data but they don't have the algorithms.
> 
> They have the only self-driving tech that can scale. Waymo and Baidu are massively behind.
> 
True they have some advantages in their business model. The assumption that they are the firm that will dominate is more speculative and doesn't make sense given the level of competition in the self-driving challenge. There are likely many winners and it's unlikely to be a winner-takes-all market in any case. This is **not** a strong reason for my scepticism though. It's more so the price action and the mania behind it. 

It seems to be everyone's favourite stock, particularly among retail investors, so that is a red flag in itself when you look at some of the broader market indicators. If it's everyone's favourite stock, you are buying late and that train has left the station. Musk has [publicly stated](https://twitter.com/elonmusk/status/1256239815256797184) that he thinks the price is too high and the stock is now 5 times higher than it was at that time. The last 12 months cannot justify that movement in valuation. 

> > 1. They have no ride-hailing business. If they were to create one, they already have competition so to assume they would instantly win the whole market is a joke.
> 
> You can adjust for 0% ride hailing. It doesn't impact the bear case that much.
> 
Okay

> > 1. Yields will continue to rise. No amount of QE will stop the bubble from imploding and Ark investors will be left holding the bag.
> 
> Yields have no impact on TSLA fundamentals. They already have enough cash to weather the storm. They're not in a credit crunch like they were in 2008, and even then they survived.
> 
The problem is that as yields rise, investors prefer risk-free assets and this leads to liquidity leaving the stonk market. If you've noticed the inverse relationship between the 10Y and NASDAQ lately, you will see why.

> > 1. Even if I'm wrong about 11 and bond yields stay below 3%, TSLA still has a P/E ratio of over 1000. An insane amount for any company and especially with slowing growth.
> 
> Amazon had a P/E ratio of 3000. P/E doesn't mean anything if you knew how business accounting works.
> 
Yes, I'm sure you know more about accounting than Jim Chanos.

> > 1. There has **never** in history been a bull market that has gone on for this long that HASN'T resulted in a 55% or more **CRASH** in the S&P. This one will rival '29.
> 
> Market timing doesn't work. Sorry.
> 
Typical fallacy. There have been many cases of timing the market. Bill Ackman did it quite well [recently](https://www.marketwatch.com/story/bill-ackmans-single-best-trade-of-all-time-turned-27-million-into-2-6-billion-now-hes-trying-it-again-11605106606).

> > Tell me the difference between Cathie's portfolio and a 12 year old with a Robinhood account.
> 
> 12 year olds can't open Robinhood accounts.
> 
But does that [stop them](https://twitter.com/TikTokInvestors/status/1359976699715325953)?

> > TSLA is going to $300 📉 along with all the other bubble stocks in Ark's ETFs.
> 
> You should short it then. Good luck!
> 
I've been short since $810. Other companies I am short: Peloton Interactive, Pinterest and Nikola

> Posts like these make me happy. It means I'm still early into TSLA.
> 
> Have a nice day!
"It means I'm still early into TSLA" - No you are quite late into it. Depends on your time horizon since it's very easy for you to claim your time horizon is 30 years or 50 years. It may be a long time before the stock reaches new all time highs. With the reopening and summer selloff, I do think we will see liquidity leaving the markets and being put into the real economy. Most of the retail investors we see are short-term speculators looking to make a quick buck.
